src/Arcus.EventGrid.Publishing/EventGridPublisher.cs

```diff
@@ -89,14 +89,37 @@ public async Task PublishRawAsync(string eventId, string eventType, string event
 
             var rawEvent = new RawEvent(eventId, eventType, eventBody, eventSubject, dataVersion, eventTime);
 
-            IEnumerable<RawEvent> eventList = new List<RawEvent>
-            {
-                rawEvent
-            };
+            IEnumerable<RawEvent> eventList = new [] { rawEvent };
```

Comment:
Since we're adding `PublishRawAsync` it might be better to just call that method instead

Reply:
Yep, done.